doc/browsing_cubes/intro.rst

```diff
@@ -1,9 +1,54 @@
 Browsing Cubes
 ==============
 
-This is a description of how you can use the cube_browser package to look at cubes and make plots and sliders without having to make any widgets.
+The Cube Browser package offers an API which you can use to visualize your cubes in a Jupyter notebook.
+It utilizes the Python, Matplotlib, Iris and Cartopy libraries while offering the additional functionality of plotting an entire cube without having to slice it.
```

Comment:
Python isn't a library; it's a programming language. How about: "It utilises Python and the Matplotlib, Iris and Cartopy libraries while..."